### DISCUSSION TITLE
Mac M1 build 怎么运行啊。（已解决）

### DISCUSSION THREAD

**Discussion by hudongyue1** [BoomingTech/Piccolo#441] [Q&A] [answered]
我可以成功build。但是不知道怎么运行Piccolo引擎。也没有在文档里找到说明。求帮助。
<img width="1461" alt="WeChat3484519854f3634e8cf6552b458e73e5" src="https://user-images.githubusercontent.com/54801900/228314028-fa68d125-5197-45f5-8cd3-07264b0832e1.png">

**Comment by ShenMian** [comment, marked as answer]
在项目根目录下的 `build` [^1]文件夹里, 可执行文件的名称应该包含了 `PiccoloEditor` [^2]. 可以搜索一下.  
一个可能的相对路径: `build\engine\source\editor\Release\PiccoloEditor`.  

[^1]: https://github.com/BoomingTech/Piccolo/blob/65379de9f67fe9c1d809e925c459d397e680ea8a/build_macos.sh#L31
[^2]: https://github.com/BoomingTech/Piccolo/blob/65379de9f67fe9c1d809e925c459d397e680ea8a/engine/source/editor/CMakeLists.txt#L1

**Reply by hudongyue1** [reply]
谢谢大佬，我找到了。

**Reply by shishouheng** [reply]
大佬能教教我怎么在m芯片build吗😭😭

**Reply by ShenMian** [reply]
@shishouheng 你可以发布一个新的 Discussion, 描述一下你遇到的问题.
